apps/google-analytics-4/frontend/src/components/config-screen/GoogleAnalytics.styles.ts

```diff
@@ -9,7 +9,7 @@ export const styles = {
     margin: '0 auto',
     marginTop: tokens.spacingXl,
     padding: `${tokens.spacingXl} ${tokens.spacing2Xl}`,
-    maxWidth: tokens.contentWidthText,
+    maxWidth: '900px',
```

Comment:
Only available widths from tokens are here. But neither is really that good for this page. Should I contact the forma team for this?
    
/**
     * @description 1280px
     */
    "contentWidthDefault": string,
    /**
     * @description 768px
     */
    "contentWidthText": string,

Reply:
this seems pretty specific, i wouldn't bother them. fine to leave as custom value